src/Process/3dTilesProcessing.js

```diff
@@ -88,6 +132,20 @@ export function pre3dTilesUpdate(context, layer) {
     // const HFOV = 2.0 * Math.atan(Math.tan(radAngle * 0.5) / context.camera.ratio);
     const HYFOV = 2.0 * Math.atan(Math.tan(radAngle * 0.5) * hypotenuse / context.camera.width);
     context.camera.preSSE = hypotenuse * (2.0 * Math.tan(HYFOV * 0.5));
+
+    // once in a while, garbage collect
+    if (Math.random() > 0.98) {
+        // Browse
+        const now = Date.now();
+
+        for (const elt of layer._cleanableTiles) {
+            if ((now - elt.cleanableSince) > (layer.cleanupDelay || 1000)) {
+                cleanup3dTileset(layer, elt);
+            }
+        }
+        layer._cleanableTiles = layer._cleanableTiles.filter(n => (layer.tileIndex.index[n.tileId].loaded && n.cleanableSince));
```

Comment:
See previous comment. The filter condition is always false, since `layer.tileIndex.index[n.tileId].loaded` is either undefined or false.

Reply:
Ok, saw your other PR, this one should come afterwards.

Reply:
Ah yes I forgot to port the `.loaded = true` to this PR.